views/update/review-beneficial-owner-individual.html

```diff
@@ -57,8 +57,11 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">Review the individual beneficial o
         {% set usual_residential_address_postcode = usual_residential_address.postcode %}
       {% endif %}
 
```

Comment:
Service address will need to be populated when the page loads as it is public data

Reply:
usual residential address is what we populate and this is requirement on review beneficial owner individual ticket